src/main/java/org/uniovi/i3a/incimanager/rest/RESTController.java

```diff
@@ -41,23 +40,20 @@
 
 	@RequestMapping(value = "/sensor-feed", method = RequestMethod.POST, consumes = {
 			MediaType.APPLICATION_JSON_VALUE }, produces = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseEntity<String> processSensorRequest( @RequestBody Map<String, Object> payload ) {
+	public ResponseEntity<Object> processSensorRequest( @RequestBody Map<String, Object> payload ) {
```

Comment:
The response entity must be an String to return a JSON as a String. JSONObject is not serializable.